README.md

```diff
@@ -1,14 +1,23 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+# SalesNote
+
+[![CI Status](https://github.com/AY2122S1-CS2103T-W08-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-W08-3/tp/actions)
+
+**SalesNote** helps Singapore-based tailors to manage local sales and clients, so that they can focus on producing great work. 
+For example, this product could help them to remember the dimensions and orders of their clients. 
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+### Example usage:
+* keeps track of all clients' details
+* adds a client to the SalesNote
+* deletes a client from the SalesNote
+* keeps track of all sales orders' details
+* adds and deletes sales orders
+* mark the sales orders as done
+
+
+### Project pages site
+The project pages site can be accessed **[here](https://ay2122s1-cs2103t-w08-3.github.io/tp/)**.
```

Comment:
Just a small detail; I think its a little more correct to write Project Page Site without the s for both lines?